src/Features/Core/Portable/FeaturesResources.resx

```diff
@@ -697,11 +697,11 @@ Do you want to continue?</value>
   <data name="UseTheNavigationBarToSwitchContext" xml:space="preserve">
     <value>You can use the navigation bar to switch context.</value>
   </data>
-  <data name="SuppressWithGlobalSuppressMessage" xml:space="preserve">
-    <value>with SuppressMessageAttribute (separate file)</value>
+  <data name="SuppressWithPragma" xml:space="preserve">
+    <value>in Source</value>
   </data>
-  <data name="SuppressWithLocalSuppressMessage" xml:space="preserve">
-    <value>with SuppressMessageAttribute</value>
+  <data name="SuppressWithGlobalSuppressMessage" xml:space="preserve">
+    <value>in Suppression File</value>
```

Comment:
New names were in fact suggested and finalized by Mark, Sri, MikeFanning and Joe :) They also felt 2 alternatives for emitting suppress message attributes was redundant and got it down to 1.